File: 1684-count-the-number-of-consistent-strings/1684-count-the-number-of-consistent-strings.java
class Solution {
    public int countConsistentStrings(String allowed, String[] words) {
        boolean isAllowed[]=new boolean[26];
        for(int i=0;i<allowed.length();i++){
            isAllowed[allowed.charAt(i)-'a']=true;
        }
        int count=0;
        for(String word:words){
            boolean consistent=true;
            for(int i=0;i<word.length();i++){
                if(!isAllowed[word.charAt(i)-'a']){
                    consistent=false;
                    break;
                }
            }
            if(consistent){
                count++;
            }
        }
        return count;
        
    }
}

File: 1684-count-the-number-of-consistent-strings/1684-count-the-number-of-consistent-strings.py
class Solution:
    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        is_allowed=[False]*26
        for char in allowed:
            is_allowed[ord(char)-ord('a')]=True
        count=0
        for word in words:
            consistent=True
            for char in word:
                if not is_allowed[ord(char)-ord('a')]:
                    consistent=False
                    break
            if consistent:
                count+=1
        return count
            
        